File: src/main/java/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedPattern/BlockAdvancedPattern.java
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedPattern;

import com.extendedae_plus.common.init.ModItems;
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase;
import net.minecraft.world.item.Item;

public class BlockAdvancedPattern extends BlockAssemblerMatrixBase<BlockEntityAdvancedPattern> {
    @Override
    public Item getPresentItem() {
        return ModItems.CoreAdvancedPattern.get();
    }
}

File: src/main/kotlin/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedPattern/BlockAdvancedPattern.kt
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedPattern

import com.extendedae_plus.common.init.EAEPItems
import com.fish.fishlib.util.extension.invoke
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase
import net.minecraft.world.item.BlockItem

class BlockAdvancedPattern : BlockAssemblerMatrixBase<TileAdvancedPattern>() {
    override fun getPresentItem(): BlockItem = EAEPItems.CoreAdvancedPattern()
}
